WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SitePreviewViewModel.kt

```diff
@@ -318,7 +318,13 @@ class SitePreviewViewModel @Inject constructor(
         object SitePreviewFullscreenProgressUiState : SitePreviewUiState(
                 fullscreenProgressLayoutVisibility = true
         ) {
-            const val loadingTextResId = R.string.notification_new_site_creation_creating_site_subtitle
+            val loadingTextResIds: List<Int> = listOf(
+                    R.string.notification_new_site_creation_creating_site_dynamic_subtitle_1,
```

Comment:
We might consider using `UIString` wrapper for consistency with other VMs. Wdyt?

Reply:
More than happy to follow existing conventions :)